src/include/server/mir/shell/idle_handler.h

```diff
@@ -52,6 +52,13 @@ class IdleHandler : public ObserverRegistrar<IdleHandlerObserver>
     /// is sent the display is never turned off or dimmed, which is the default.
     virtual void set_display_off_timeout(std::optional<time::Duration> timeout) = 0;
 
+    /// Duration Mir will remain idle before the display is turned off after the session is locked. The display may dim
+    /// some time before this. If not nullopt, this idle timeout takes precedence over the timeout set with
+    /// set_display_off_timeout while the session is locked and remains idle. If Mir becomes active after the session
+    /// is locked, this timeout is disabled. If the session is already locked when this function is called, the new
+    /// timeout will only take effect after the next session lock.
+    virtual void set_display_off_timeout_on_lock(std::optional<time::Duration> timeout) = 0;
```

Comment:
[Aside] This is breaking the mirserver ABI. We need to bump accordingly (either now or before 2.19)

Don't worry about it here - there's other stuff breaking things in the pipeline

Reply:
Oops, I didn't notice it before because the symbols checking tool doesn't detect changes in the vtable.
If it's just a matter of bumping `MIR_VERSION_MINOR` and `MIRSERVER_ABI`, we can do it now. Is it enough to follow the [how-to](https://github.com/canonical/mir/blob/main/doc/sphinx/how-to/how-to-update-symbols-map.md)?